integration/indexes_compat_test.go

```diff
@@ -751,3 +751,124 @@ func TestDropIndexesCommandCompat(t *testing.T) {
 		})
 	}
 }
+
+func TestCreateIndexesUniqueCompat(t *testing.T) {
+	setup.SkipForTigrisWithReason(t, "Indexes creation is not supported for Tigris")
+
+	t.Parallel()
+
+	for name, tc := range map[string]struct { //nolint:vet // for readability
+		models    []mongo.IndexModel // required, index to create
+		insertDoc bson.D             // required, document to insert for uniqueness check
+		new       bool               // optional, insert new document before check uniqueness
+
+		skip string // optional, skip test with a specified reason
+	}{
+		"ExistingFieldIndex": {
```

Comment:
Could you test the case with setting unique to `_id` please.

Reply:
Added.

Reply:
Added a test to `TestCreateIndexesInvalidSpec`.

Reply:
Great thanks!